iree/compiler/Dialect/VMLA/IR/test/ops.mlir

```diff
@@ -74,24 +74,19 @@ func @vmla_batch_matmul_pseudo(%lhs : tensor<32x256x128xf32>,
 // CHECK-SAME: %[[DST:[a-zA-Z0-9]+]]
 // CHECK-SAME: %[[DST_SHAPE:[a-zA-Z0-9]+]]
 func @vmla_batch_matmul(%lhs : !vmla.buffer,
-                        %lhs_shape : !shapex.ranked_shape<[32,10,784]>,
+                        %lhs_shape : !shapex.ranked_shape<[8,10,64]>,
                         %rhs : !vmla.buffer,
-                        %rhs_shape : !shapex.ranked_shape<[32,1,784]>,
+                        %rhs_shape : !shapex.ranked_shape<[8,1,64]>,
                         %dst : !vmla.buffer,
-                        %dst_shape : !shapex.ranked_shape<[32,1,10]>) {
+                        %dst_shape : !shapex.ranked_shape<[8,1,10]>) {
   // CHECK:      vmla.batch.matmul
-  // CHECK-SAME:   %[[LHS]], %[[LHS_SHAPE]],
-  // CHECK-SAME:   %[[RHS]], %[[RHS_SHAPE]],
-  // CHECK-SAME:   %[[DST]], %[[DST_SHAPE]] :
-  // CHECK-SAME: (f32, !shapex.ranked_shape<[32,10,784]>
-  // CHECK-SAME:  f32, !shapex.ranked_shape<[32,1,784]>
-  // CHECK-SAME:  f32, !shapex.ranked_shape<[32,1,10]>)
-  vmla.batch.matmul %lhs, %lhs_shape,
-                    %rhs, %rhs_shape,
-                    %dst, %dst_shape :
-                   (f32, !shapex.ranked_shape<[32,10,784]>,
-                    f32, !shapex.ranked_shape<[32,1,784]>,
-                    f32, !shapex.ranked_shape<[32,1,10]>)
+  // CHECK-SAME: %[[LHS]](%[[LHS_SHAPE]] !shapex.ranked_shape<[8,10,64]>) : f32,
+  // CHECK-SAME: %[[RHS]](%[[RHS_SHAPE]] !shapex.ranked_shape<[8,1,64]>) : f32,
+  // CHECK-SAME: out
+  // CHECK-SAME: %[[DST]](%[[DST_SHAPE]] !shapex.ranked_shape<[8,1,10]>) : f32
+  vmla.batch.matmul %lhs(%lhs_shape !shapex.ranked_shape<[8,10,64]>) : f32,
+                    %rhs(%rhs_shape !shapex.ranked_shape<[8,1,64]>) : f32,
+                    out %dst(%dst_shape !shapex.ranked_shape<[8,1,10]>) : f32
```

Comment:
In MLIR, usually a colon separates an SSA-value and its type

What about something like this?

```suggestion
  vmla.batch.matmul %lhs : f32(%lhs_shape : !shapex.ranked_shape<[8,10,64]>),
                    %rhs : f32(%rhs_shape : !shapex.ranked_shape<[8,1,64]>),
                    out %dst : f32(%dst_shape : !shapex.ranked_shape<[8,1,10]>)
```

Reply:
Yes, I missed that. I'll add it!

Reply:
Ah I was also suggesting putting the "f32" closer to the operand it's referring to

Reply:
Having the parens around the shape information directly attached to the `f32` looks a bit strange to my eyes. At that point we're very close to just interleaving the type information with the SSA values and comma delimiting them, which might look cleaner than the `f32()` syntax? At the same time I haven't seen anything interleaved like that in MLIR before. What do you think?
```mlir
%lhs(%lhs_shape : !shapex.ranked_shape<[8,4,4]>) : f32,
%rhs(%rhs_shape : !shapex.ranked_shape<[8,1,4]>) : f32,
out %dst(%dst_shape : !shapex.ranked_shape<[8,1,4]>) : f32
vs.
%lhs : f32(%lhs_shape : !shapex.ranked_shape<[8,4,4]>),
%rhs : f32(%rhs_shape : !shapex.ranked_shape<[8,1,4]>),
out %dst : f32(%dst_shape : !shapex.ranked_shape<[8,1,4]>)
vs.
%lhs : f32, %lhs_shape : !shapex.ranked_shape<[8,4,4]>,
%rhs : f32, %rhs_shape : !shapex.ranked_shape<[8,1,4]>,
out %dst : f32, %dst_shape : !shapex.ranked_shape<[8,1,4]>
```